examples/visual-tests/amp-story/amp-story-tooltip.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story"
+        src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <title>amp-story tooltip diff</title>
+    <meta name="viewport"
+          content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+      amp-story-page {
+        background: black;
+        color: white;
+        font-family: sans-serif;
+      }
+
+      .i-amphtml-story-spinner-container {
+        /* Hide the spinner for loading pages, since this causes flakes */
+        display: none;
+      }
+
+      h1.hello-world {
+        color: white;
+        text-shadow: 3px 3px rgba(0, 0, 0, 0.5);
+      }
+
+      .title-small {
+        font-family: 'Roboto Condensed', sans-serif;
+        font-weight: normal;
+        font-size: 20px;
+        text-transform: uppercase;
+        text-align: center;
+        color: white;
+      }
+    </style>
+    <script>
+      // TODO(#17108): Remove this when getMode().test is set.
+      AMP_TEST = true;
```

Comment:
My tests wouldn't work unless this was set. (e.g. `await page.tap('a.title-small.center');` wouldn't actually show the tooltip in the story).

Linking #17108

@danielrozenberg

Reply:
Fixed in #19316, rebase on latest master if you need it!

Reply:
Ok that worked. Removed it from here.